src/Tizen.MachineLearning.Service/Tizen.MachineLearning.Service/Service.cs

```diff
@@ -190,6 +190,38 @@ protected virtual void Dispose(bool disposing)
             _disposed = true;
         }
 
+        internal static TensorsInfo ConvertTensorsInfoFromHandle(IntPtr handle)
+        {
+            TensorsInfo retInfo = null;
+            MLServiceError ret = MLServiceError.None;
+
+            int rankLimit = 16;
```

Comment:
https://github.com/niley7464/TizenFX/blob/080634f97cd33da9ebdd28cf5626767ed8a39c99/src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Commons.cs#L270

RankLimit is internal value..